persistent-sql-lifted/library/Database/Persist/Sql/Lifted/Expression/ArrayAggregate/PostgreSQL.hs

```diff
@@ -0,0 +1,89 @@
+module Database.Persist.Sql.Lifted.Expression.ArrayAggregate.PostgreSQL
+  ( AggMode (..)
+  , arrayAggDistinct
+  , arrayAgg
+  , arrayAggWith
+  , arrayRemove
+  , arrayRemoveNull
+  , maybeArray
+  , arrayAggById
+  , arrayAggByIdMaybe
+  , arrayAggByMaybe
+  , arrayAggBy
+  ) where
+
+import Prelude
+
+import Database.Esqueleto.Experimental
```

Comment:
Yeah... we are now very far from the idea of "this is `persistent-sql` but lifted", which is what the package name implies.

Is this our first introduction of general Esqueleto functionality? If so, I'd probably avoid it since we haven't actually ruined the name yet. If not, then oh well keep on keepin on.

Reply:
Yeah the ship has already sailed. Maybe it can be made more beautiful in a v1.0 someday